src/client/config_info.h

```diff
@@ -151,7 +146,11 @@ struct FailureRequestOption {
     uint64_t chunkserverMaxRPCTimeoutMS = 64000;
     uint64_t chunkserverMaxRetrySleepIntervalUS = 64ull * 1000 * 1000;
     uint64_t chunkserverMinRetryTimesForceTimeoutBackoff = 5;
-    uint64_t chunkserverMaxRetryTimesBeforeConsiderSuspend = 20;
+
+    // When a request remains outstanding beyond this threshold, it is marked as
+    // a slow request.
+    // Default: 45s
+    uint32_t chunkserverSlowRequestThresholdMS = 45 * 1000;
```

Comment:
Should it be set to a flag so that the corresponding threshold can be dynamically modified?

Reply:
This configuration will not be modified frequently